src/main/java/Duke.java

```diff
@@ -1,10 +1,104 @@
+import java.util.Scanner;
+import java.util.Arrays;
 public class Duke {
+    public static final int MAX_TASK_NUM = 100;
     public static void main(String[] args) {
+        showGreetings();
+        processInputs();
+        showGoodbye();
+    }
+
+    private static void processInputs() {
+        Scanner in = new Scanner (System.in);
+        Task[] storeValues = new Task[MAX_TASK_NUM];
+        int counter = 0;
+
+        String line = in.nextLine();
+        while (!line.equals("bye")) {
+            if (line.equals("list")) {
+                printList(storeValues, counter);
+            } else if (line.startsWith("mark ")) {
+                markItem(storeValues, line);
+            } else if (line.startsWith("unmark ")) {
+                unmarkItem(storeValues, line);
+            } else {
+                counter = addTask(storeValues, counter, line);
+            }
+            line = in.nextLine();
+        }
+    }
+
+    private static int addTask(Task[] storeValues, int counter, String line) {
+        formattingLine();
+        System.out.println("added: " + line);
+        formattingLine();
+        //store value in line into list
```

Comment:
Need to add whitespace at the start of the comment, and begin with comment with `Stores` instead of `store`